shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-env/src/test/java/org/apache/shardingsphere/test/integration/env/container/atomic/adapter/impl/ShardingSphereProxyClusterContainer.java

```diff
@@ -77,44 +76,13 @@ public ShardingSphereProxyClusterContainer withAgent(final String agentHome) {
     protected void configure() {
         withExposedPorts(3307);
         mapConfigurationFiles();
-        if (Strings.isNullOrEmpty(module)) {
-            setWaitStrategy(new JDBCConnectionWaitStrategy(() -> DriverManager.getConnection(DataSourceEnvironment.getURL(databaseType,
-                    getHost(), getMappedPort(3307), scenario), "proxy", "Proxy@123")));
-        }
-        if ("scaling".equalsIgnoreCase(module)) {
-            scalingConfigure();
-        }
-    }
-    
-    private void scalingConfigure() {
-        if (DatabaseTypeUtil.isPostgreSQL(databaseType)) {
-            setWaitStrategy(
-                    new JDBCConnectionWaitStrategy(() -> DriverManager.getConnection(DataSourceEnvironment.getURL(databaseType, getHost(), getMappedPort(3307), "postgres"), "proxy", "Proxy@123")));
-        } else if (DatabaseTypeUtil.isMySQL(databaseType)) {
-            setWaitStrategy(new JDBCConnectionWaitStrategy(() -> DriverManager.getConnection(DataSourceEnvironment.getURL(databaseType, getHost(), getMappedPort(3307), ""), "proxy", "Proxy@123")));
-        }
+        // TODO in suite, dataSourceName should be scenario
+        setWaitStrategy(new JDBCConnectionWaitStrategy(() -> DriverManager.getConnection(DataSourceEnvironment.getURL(databaseType,
+                getHost(), getMappedPort(3307), adaptorContainerConfiguration.getWaitStrategyInfo().get("dataSourceName")), "proxy", "Proxy@123")));
     }
     
     private void mapConfigurationFiles() {
```

Comment:
It is better to rename to mountConfigurationFiles

Reply:
done